src/components/Panel/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Dialog, Transition } from "@headlessui/react";
+import React, { FC } from "react";
+
+const Panel: FC<any> = (props) => (
```

Comment:
nit: `Panel` doesn't need to be in a separate folder, e.g. no need for a `/components/Panel/index.tsx` structure. It can just be placed in a file `/components/Panel.tsx`.